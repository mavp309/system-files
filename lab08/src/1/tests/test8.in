3.33333 4.44444
-5.55555 -6.66666
