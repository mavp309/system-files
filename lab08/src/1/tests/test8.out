Sum: -2.2222+-2.2222j
Difference: 8.8889+11.1111j
Product: 11.1111+-46.9135j
Division: -0.6393+-0.0328j
